libcxx/test/std/algorithms/alg.nonmodifying/alg.count/ranges.count.pass.cpp

```diff
@@ -251,7 +262,7 @@ constexpr bool test() {
       assert(ret == 1);
     }
     {
-      int a[] = {5, 5, 4, 3, 2, 1};
+      int a[]                                      = {5, 5, 4, 3, 2, 1};
       auto range = std::ranges::subrange(DiffTypeIterator(a), DiffTypeIterator(a + 6));
       std::same_as<signed char> decltype(auto) ret = std::ranges::count(range, 4);
```

Comment:
```suggestion
      int a[]                                      = {5, 5, 4, 3, 2, 1};
      auto range = std::ranges::subrange(DiffTypeIterator(a), DiffTypeIterator(a + 6));
      std::same_as<signed char> decltype(auto) ret = std::ranges::count(range, 4);
```
This appears to be a bug in clang-format, which requires aligning `=` signs on two non-adjacent lines.